plugins/arcgis/web-app/projects/main/src/lib/arc-event/arc-event.component.scss

```diff
@@ -1,69 +1,16 @@
-.arc-layer-form {
+.arcEvent {
     display: flex;
-    flex-direction: column;
-    min-width: 950px;
+    justify-content: space-between; 
+    align-items: center;
+    margin-top: 16px;
 }
 
-section {
-    >* {
-        margin-inline-start: 1em;
-    }
-
-    header {
-        margin-inline-start: 0;
-
-        h1,
-        h2,
-        h3,
-        h4,
-        h5,
-        h6 {
-            margin-block-end: 0;
-        }
-
-        .subheading {
-            margin-block-start: 0;
-            opacity: 0.5;
-        }
-
-        margin-block-end: 2em;
+.arc-event {
```

Comment:
I think the default angular material styling for the card header title/subtitle is fine.  Lets remove this class and the reference to it in the html template.